test/Datadog.Trace.ClrProfiler.IntegrationTests/TestCollections/StackExchangeRedisTestCollection.cs

```diff
@@ -0,0 +1,9 @@
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.TestCollections
+{
+    [CollectionDefinition(nameof(StackExchangeRedisTestCollection), DisableParallelization = true)]
```

Comment:
@zacharycmontoya
side note: probablyDisableParallelization = true` was what I was missing in my PoC 😉